src/main/java/com/atlassian/bitbucket/jenkins/internal/client/HttpRequestExecutor.java

```diff
@@ -19,7 +20,6 @@
      * @param consumer,   on successful execution, {@link Response} will be passed to consumer.
      * @param <T>,        result that consumer wish to return.
```

Comment:
How about we name these `ResponseType` rather than just `T` makes it easier to know what they are and the computer doesn't care :D

Reply:
I personally prefer a single letter, reason being, using word creates a confusion of it being a class name than being a type parameter.